inversion-api/src/main/java/io/inversion/cloud/action/security/AuthAction.java

```diff
@@ -158,12 +158,15 @@ else if (token != null && token.toLowerCase().startsWith("bearer "))
                }
                catch (Exception ex)
                {
+                  // TODO Why not just throw the 401 here?  With a context specific log?
+                  log.debug("Attempting to validate token failed {}  Continuing.", token, ex) ;
```

Comment:
I think the concept here is that it tries to verify with each secret. If one succeeds (156), then we break. Otherwise is drops in here and i guess should be ignored?